netdata-installer.sh

```diff
@@ -934,6 +934,34 @@ if [ "$have_autotools" ]; then
   fi
 fi
 
+# function to extract values from the config file
+config_option() {
+  section="${1}"
+  key="${2}"
+  value="${3}"
+
+  if [ -s "${NETDATA_PREFIX}/etc/netdata/netdata.conf" ]; then
+    "${NETDATA_PREFIX}/usr/sbin/netdata" \
+      -c "${NETDATA_PREFIX}/etc/netdata/netdata.conf" \
+      -W get "${section}" "${key}" "${value}" ||
+      echo "${value}"
+  else
+    echo "${value}"
+  fi
+}
+
+# the user netdata will run as
+if [ "$(id -u)" = "0" ]; then
+  NETDATA_USER="$(config_option "global" "run as user" "netdata")"
+  ROOT_USER="root"
+else
+  NETDATA_USER="${USER}"
+  ROOT_USER="${USER}"
+fi
+NETDATA_GROUP="$(id -g -n "${NETDATA_USER}")"
+[ -z "${NETDATA_GROUP}" ] && NETDATA_GROUP="${NETDATA_USER}"
+echo >&2 "Netdata user and group is finally set to: ${NETDATA_USER}/${NETDATA_GROUP}"
```

Comment:
```suggestion
echo >&2 "Netdata user and group set to: ${NETDATA_USER}/${NETDATA_GROUP}"
```

More concise message. ‘finally’ sounds unnatural here, and without it the ‘is’ is not needed.